src/openrct2/object/BannerObject.cpp

```diff
@@ -40,6 +42,18 @@ void BannerObject::ReadLegacy(IReadObjectContext * context, IStream * stream)
     {
         context->LogError(OBJECT_ERROR_INVALID_PROPERTY, "Price can not be free or negative.");
     }
+
+    // Add banners to 'Signs and items for footpaths' group, rather than lumping them in the Miscellaneous tab.
+    // Since this is already done the other way round for original items, avoid adding those to prevent duplicates.
+    const std::string identifier = GetIdentifier();
+    uint8 source = object_entry_get_source_game(object_list_find_by_name(identifier.c_str()));
```

Comment:
Probably worth checking for null on `object_list_find_by_name(identifier.c_str())`